docs/pages/attributesJSON.rst

```diff
@@ -0,0 +1,194 @@
+	- Describes the **up** direction for the stage.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the stage.
+"origin"
+	- 3-vector
+	- Describes the **origin** of the stage, for alignment purposes.
+
+Stage Physics and Object-related Parameters
+-------------------------------------------
+
+Below are stage-specific physical and object-related quantities.  These values will override similarly-named values specified in the Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the stage.
+"gravity"
+	- 3-vector
+	- Gravity to use for physical modeling.
+"margin"
+	- double
+	- Distance margin for collision calculations.
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+"units to meters"
+	- double
+	- The conversion of given units to meters.
+"requires lighting"
+	- boolean
+	- Whether or not the stage should use lighting.
+
+`Object Attributes`_
+====================
+Object Attributes templates hold descriptive information for type of object that can be loaded into Habitat.  These files should be named in the format :
+
+ 	<objectname>.phys_properties.json
+
+An example of an appropriate configured file can be found below :
+
+.. include:: ../../data/test_assets/objects/donut.phys_properties.json
+    :code: json
+
+Object Mesh Handles And Types
+-----------------------------
+
+Below are the handles and descriptors for various mesh assets used by an object.
+
+"render mesh"
+	- string
+	- The name of the file describing the render mesh to be used by the object.
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the object.
+
+Object Frame and Origin
+-----------------------
+
+The tags below are used to build a coordinate frame for the object, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.  The object's COM is used as its origin.
+
+"up"
+	- 3-vector
+	- Describes the **up** direction for the object.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the object.
+
+
+Below are object-specific physical quantities.  These values will override similarly-named values specified in a Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the object.
+"margin"
+	- double
+	- Distance margin for collision calculations.
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+"units to meters"
+	- double
+	- The conversion of given units to meters.
+"requires lighting"
+	- boolean
+	- Whether or not the object should use lighting.
+"mass"
+	- double
+	- The mass of the object, for physics calculations.
+"inertia"
+	- 3-vector
+	- The values of the diagonal of the inertia matrix for the object.
```

Comment:
Maybe add: "If not provided, will computed automatically from the object's mass and bounding box."